.github/workflows/ci.yml

```diff
@@ -33,3 +35,33 @@ jobs:
     - run: npm test
       env:
         CI: true
+
+  job_2:
+
+    name: Deploy to gh-pages when push to master branch
+
+    needs: job_1
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+        with:
+          persist-credentials: false
+
+      - name: Install and Build
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+        run: |
+          npm install
+          npm run build-production --if-present
```

Comment:
do we need to build docs? `npm run build-docs`?

Reply:
@huningxin OK, I will add this step.